static/js/components/ProfileImageUploader.js

```diff
@@ -74,7 +74,7 @@ const ProfileImageUploader = ({
       </Button>,
       <Button
         type='button'
-        className='primary-button save-button'
+        className={photo ? 'primary-button save-button' : 'secondary-button save-button disabled'}
```

Comment:
this is a bit redundant. since we always want `.save-button` why not do something like:

```js
className={`save-button ${photo ? 'secondary-button disabled' : 'primary-button'}`}
```